lib/account/pages/account_page.dart

```diff
@@ -68,6 +74,12 @@ class _AccountPageState extends State<AccountPage> {
               ),
             ),
           ),
+          actions: <Widget>[
+            TextButton(
+              child: const Text("Ok"),
+              onPressed: () => Navigator.of(context).pop(),
+            ),
+          ],
         ),
       ),
     );
```

Comment:
I can't comment on collapsed lines, but the "Sign out" Button does not seem to be clickable currently